buildSrc/src/main/groovy/bio.terra.landingzone.java-spring-conventions.gradle

```diff
@@ -4,8 +4,6 @@ plugins {
     id 'org.springframework.boot'
 }
 
-// Spring Boot 3.1.2 pulls in postgresql 42.6.0, vulnerable to CVE-2024-1597
-ext['postgresql.version'] = '42.7.2'
```

Comment:
Spring Boot 3.2.3 gets us to 42.6.1, which is past the vulnerability.  In general, these pins are hard to maintain so my preference is to remove them whenever possible.